src/runtime-rs/crates/hypervisor/src/qemu/inner.rs

```diff
@@ -68,6 +68,8 @@ impl QemuInner {
         // descriptor needs to stay open until the qemu process launches.
         // This is why we need to store it in a variable at this scope.
         let mut _vhost_fd = None;
+        let mut _tun_fds: Vec<std::fs::File> = Vec::new();
+        let mut _vhost_fds: Vec<std::fs::File> = Vec::new();
```

Comment:
We don't need (and probably don't want) to know about the underlying type of fd. We just want them to stay open in the current scope. I'd prefer to have a single `_fds_for_qemu`  array to hold them all, for the sake of clarity. The comment above needs to be adapted as well.